views/demo.pug

```diff
@@ -4,13 +4,13 @@ block content
 	.container-fluid
 		.row
 			.col-sm-3.col-lg-2
-				nav.side-navbar.navbar-default.navbar-fixed-side
+				.side-navbar
 					ul
-						li
-							a(href="#") Link-1
-						li
-							a(href="#") Link-2
-						li
-							a(href="#") Link-3
+						a(href="#")
+							li Link-1
+						a(href="#")
```

Comment:
this is invalid html you can not wrap a block level element with a non block level element. If you want the whole thing to be the link you style the a to act like a block and define all your layout on the a. you actually should never have an styling on li. You always want to style the a